src/structures/WelcomeScreen.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { default: Collection } = require('@discordjs/collection');
+const Base = require('./Base');
+const WelcomeChannel = require('./WelcomeChannel');
+
+/**
+ * Represents a welcome screen.
+ */
```

Comment:
```suggestion
/**
 * Represents a welcome screen.
 * @extends {Base}
 */
```
This class also needs to be added to the exports